kubernetes/create-cluster.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+datacenterID=$(ionosctl datacenter list -ojson | jq -r '.Resources[] | select(.properties.name == "joe-test") | .id')
+
+# ionosctl image list | grep "Image\|ubuntu-22"
+
+for i in {1..3}; do
+  export serverName="controlplane-$i"
+  export volName="vol-$serverName"
+  ionosctl volume create --datacenter-id "$datacenterID" --name "$volName" --image-alias ubuntu:22.04 --size 80 --type SSD --ssh-key-paths /home/jmanser/.ssh/id_rsa.pub
```

Comment:
Things like these do not belong in a public repository IMO (I mean a direct path to the SSH key on *your* machine)

Reply:
ahh wanted to do a env var there and forgot, sorry.